VaticanBAV/et/BAVet120.xml

```diff
@@ -0,0 +1,189 @@
+  <height>224</height>
+  <width>166</width>
+  </dimensions>
+</extent>
+<collation>
+<list>
+  <item xml:id="q1" n="A"><dim unit="leaf">2</dim></item>
+<item xml:id="q2" n="1"><num value="6">፮</num><dim unit="leaf">10</dim><locus from="3r" to="12v"></locus></item>
+<item xml:id="q3" n="2"><num value="7">፯</num><dim unit="leaf">10</dim><locus from="13r" to="22v"></locus></item>
+<item xml:id="q4" n="3"><num value="8">፰</num><dim unit="leaf">10</dim><locus from="23r" to="32v"></locus></item>
+<item xml:id="q5" n="4"><num value="9">፱</num><dim unit="leaf">10</dim><locus from="33r" to="42v"></locus></item>
+<item xml:id="q6" n="5"><num value="1">፩</num><dim unit="leaf">10</dim><locus from="43r" to="52v"></locus></item>
+<item xml:id="q7" n="6"><num value="2">፪</num><dim unit="leaf">10</dim><locus from="53r" to="62v"></locus></item>
+<item xml:id="q8" n="7"><num value="3">፫</num><dim unit="leaf">10</dim><locus from="63r" to="72v"></locus></item>
+<item xml:id="q9" n="8"><num value="4">፬</num><dim unit="leaf">10</dim><locus from="73r" to="82v"></locus></item>
+<item xml:id="q10" n="9"><num value="5">፭</num><dim unit="leaf">10</dim><locus from="83r" to="92v"></locus></item>
+<note>The quires are numbered. Quires no. 6-9 (ff. 3-42) contain the text of
+  Śәna fәṭrat, quires no. 1-5 (ff. 43-92) contain ʾAmmǝstu ʾaʿǝmāda mǝśṭir and
+  blank pages</note>
+</list>
+</collation>
+</supportDesc>
+<layoutDesc><layout writtenLines="24"></layout></layoutDesc>
+</objectDesc>
+
+<handDesc>
+<handNote xml:id="h1" script="Ethiopic">
+<seg type="ink"> Black, red</seg>
+<seg type="rubrication">Titles</seg>
+<date when="1923"></date>
+</handNote>
+</handDesc>
+
+<additions>
+<list>
+<item xml:id="a1">
+<locus target="42v"></locus>
+<desc type="Subscription"></desc>
+<q xml:lang="gez">ዝ፡ መጽሐፍ፡ ዘ<placeName ref="LOC1713Bagemd">በጌምድር፡</placeName>
+ <placeName ref="LOC3056Farta">ፋርጣ፡</placeName> ዓዲኮ፡ <persName ref="PRS12979GabrielFA">ገብርኤል</persName>፡
+ ዘወሀቦ፡ <persName ref="PRS12978GabraSelassie">ገብረ፡ ሥላሴ፡</persName> እኁሁ፡
+  ለ<persName ref="PRS12980WaldaGiyrgis">ወልደ፡ ጊዮርጊስ፡ ዘውእቱ፡ ወርቄ፡ መርሻ</persName>፡ ሳምን፡ ትውልድ፡ እምነ፡ አቅኚው፡ <persName ref="PRS12991BegaKrestos">በጎ፡
+  ክርስቶስ</persName>፨ <persName ref="PRS12993ShumeKrestos">ሹመ፡ ክርስቶስ</persName>።
+  <persName ref="PRS12992WalattaKrestos">ወለተ ማርያም</persName>፨
+  <persName ref="PRS12995GetaWeldye">ጌታ፡ ወልድየ</persName>፨
+  <persName ref="PRS12996AdadaMaryam">ዓፀደ፡ ማርያም</persName>፨
+  <persName ref="PRS12997Altasab">አልታሰብ</persName>፨
+<persName ref="PRS12998Werqe">ወርቄ</persName>፨
+<persName ref="PRS12999GebSelMer">ገብረ፡ ሥላሴን፡ መርሻን</persName> ። ። ።
+ ያቡቲ፡ አቅኚው፨ <persName ref="PRS13000DemeKrestos">ደመ፡ ክርስቶስ</persName>፨
+  ዘሜ። <persName ref="PRS13007WalattaNabiyat">ወለተ፡
+  ነቢያት</persName>። <persName ref="PRS13008Akal">አካል</persName>፨
+  <persName ref="PRS13009Gezot">ግዛት</persName>፨
+  <persName ref="PRS13005WalattaSeyon"> ወለተ፡ ጽዮን</persName>፨ <persName ref="PRS13010Warqe">ወርቄን፡</persName>
+  ስመ፡ ጥምቀቱ፡ ወልደ፡ ሚካኤል፤ እረፍቱኒ፡
```

Comment:
Could any roles of these persons be idetified? Next to ref in the opening tag you can add an atribute role with different values like owner, donor, and so on

Reply:
One is an owner, one is a donor, the rest is in my opinion just genealogy

Reply:
So, you can add those roles to these persons. It is done like that: <persName ref="...." role="scribe">...., owner instead of scribe respectively. There will be a prod down with possible rolenames